src/components/Update.js

```diff
@@ -62,6 +63,18 @@ class Update extends React.Component {
     this.state.mode === 'edit' ? this.cancelEdit() : this.edit();
   }
 
+  async deleteUpdate() {
+    if (!confirm('😱 Are you really sure you want to delete this update?'))
+      return;
+
+    try {
+      await this.props.deleteUpdate(this.props.update.id);
+      Router.pushRoute(`/${this.props.collective.slug}`);
```

Comment:
To prevent a full page reload, using the named route and pass the slug as a param:

```es6
Router.pushRoute('collective', { slug: this.props.collective.slug });
```

Can you confirm @znarf?

Reply:
@HipsterBrown This might be equivalent. My understanding is that the Router is able to parse the URL and go to the route. Semantically, I prefer the syntax you proposed.